x-pack/plugin/esql/qa/testFixtures/src/main/resources/mv_expand.csv-spec

```diff
@@ -252,3 +252,20 @@ emp_no:integer  | first_name:keyword | generate_mv:keyword
 10001           | Georgi             | foo
 10002           | Bezalel            | foo
 ;
+
+
+// see https://github.com/elastic/elasticsearch/issues/102120
+expandAfterDuplicateAggs
```

Comment:
@luigidellaquila You might need to append `#[skip:-8.11.99]` to these two tests since they don't work with 8.11.0 or 8.11.1. Similarly to YAML tests, you can adjust the skipped version after backporting.